### PR TITLE
Add data sources

### PR DESCRIPTION
**Pull request by DavidMajomi**

### CODE REVIEW

File: Src/scripts/get_stock_data.py
Comment (JesseMurah) [inline Src/scripts/get_stock_data.py:1]: i want to know whether you'll be depending on yfinance(as in current stocks) or you downloaded the stocks data as .csv format

**Review by JesseMurah — CHANGES_REQUESTED** [review]
In the `Src/scripts/get_stock_data.py` have you downloaded the datasets already or how did you get the data? From Yfinances' site or kaggle or where did you get it.
Apart from that, all the codes are good.

File: Src/scripts/get_stock_data.py
Comment (DavidMajomi) [reply Src/scripts/get_stock_data.py:1]: I scraped it from wikipedia to get all tickers in the info as csv in scrape.py. Then i loaded it as a pandas data frame to get price data for each ticker. The price data is then stored in a database.

Comment (DavidMajomi) [reply Src/scripts/get_stock_data.py:1]: So yes, I will be depending on yfinance, but only when I need to populate historical price data such as the first time training the models.

**Pull request merged**